### ISSUE TITLE
import:user [options] [--] <exportDirectory> --> <importDirectory>

### ISSUE THREAD

**Issue by mmattel** [owncloud/data_exporter#46]
**Exporting**
```
export:user [options] [--] <exportDirectory>
exportDirectory    Path to export to.
```
**Importing**
```
import:user [options] [--] <exportDirectory>
exportDirectory    Path to export directory
```
Exporting:
The description can be improved.
This should be changed to
``exportDirectory``    ``Path to the directory to export data to``

Importing:
Imho, the naming and description of the argument this is bad.
This should be changed to
``importDirectory``    ``Path to the directory to import data from``

@DeepDiver1975

**Comment by DeepDiver1975** [comment]
Yes - please adjust .....THX a lot.

Feel free to put this change right away into #45

**Comment by mmattel** [comment]
Closing, fixed via #45
